libbeat/docs/template-config.asciidoc

```diff
@@ -82,3 +82,7 @@ setup.template.overwrite: false
 setup.template.settings:
   _source.enabled: false
 ----------------------------------------------------------------------
+
+*`setup.template.append_fields`*:: A list of of fields to be added to the template and Kibana index pattern.
```

Comment:
Based on the changelog comment this should probably be marked as experimental.

Reply:
experimental flag added.